pkg/api/handlers/compat/containers_stop.go

```diff
@@ -43,7 +43,8 @@ func StopContainer(w http.ResponseWriter, r *http.Request) {
 		}
 	} else {
 		if _, found := r.URL.Query()["t"]; found {
-			options.Timeout = &query.DockerTimeout
+			timeout := uint(query.DockerTimeout)
```

Comment:
Doesn't this technically only cause a overflow?
I mean wait for `18446744073709551615` seems more than good enough for wait unlimited but a comment would help to make clear that we only set the timeout to some high value